app/controllers/users/sessions_controller.rb

```diff
@@ -1,2 +1,13 @@
 class Users::SessionsController < Devise::SessionsController
+  def new
```

Comment:
This is a little hacky... but OK.

I'd prefer to see a less hacky approach, but maybe that's not possible? Not blocking.